gm4_mountaineering/data/gm4_mountaineering/functions/climbing_effects/damage_shulkers.mcfunction

```diff
@@ -0,0 +1,7 @@
+# increase the score of each climbing shulker, and kill after reaching 6
+# @s = shulker with tag gm4_mountaineering_climb_assist
+# at @s
+# run from player_main
+
+scoreboard players add @s gm4_mountaineering_climbing_shulker_countdown 1
+execute as @s[scores={gm4_mountaineering_climbing_shulker_countdown=6..}] run function gm4_mountaineering:climbing_effects/kill_shulkers
```

Comment:
Functions have a bit of an overhead, so it'd probably be better to just run the kill and tp commands here.
```suggestion
teleport @s[scores={gm4_mountaineering_climbing_shulker_countdown=6..}] ~ -10000 ~
kill @s[scores={gm4_mountaineering_climbing_shulker_countdown=6..}]
```